src/components/NavBar/NavigationBar.tsx

```diff
@@ -1,18 +1,27 @@
 import React, {useState, useEffect} from 'react';
 import './NavigationBar.css';
 import cehatLogo from './CEHATLogo.png';
-import menuIcon from './menuIcon.png';
 import {useTranslation} from 'react-i18next';
+import {FaBars, FaGlobeAmericas} from 'react-icons/fa';
 
 function NavigationBar(): JSX.Element {
+  // State for whether to use globe or text, must be kept
+  // separate because mobile should always be text, even
+  // when nav bar is hidden
+  const [isMobile, setIsMobile] = useState(
+    window.matchMedia('(max-width: 700px)')?.matches
```

Comment:
Let's not use optional chaining here since if window.matchMedia doesn't exist, then it will default to null instead of a boolean like we'd want.